debian_install.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+NEPHOS_DIR=$(pwd)
+
+if [ `id -u` -ne 0 ]
+    then echo "Error: please launch the script as sudo!"
+    exit 1
+fi
+
+# install python3
+add-apt-repository ppa:deadsnakes/ppa -y
+apt-get update -y 
+apt-get install python3.4
+easy_install-3.4 pip
+echo "python-pip installed"
+# install pipenv
+pip3 install pipenv
+export PYTHON_BIN_PATH="$(python3 -m site --user-base)/bin"
+export PATH="$PATH:$PYTHON_BIN_PATH"
+echo "pipenv installed"
+
+apt-get install -y mailx screen
```

Comment:
This part gives me error E: Unable to locate package malix apt install screen works fine for me :confused: